lib/sqlalchemy/pool/impl.py

```diff
@@ -48,7 +48,7 @@ class QueuePool(Pool):
     that imposes a limit on the number of open connections.
 
     :class:`.QueuePool` is the default pooling implementation used for
-    all :class:`_engine.Engine` objects, unless the SQLite dialect is in use.
```

Comment:
OK like I said at #9747, this should say, "unless the SQLite dialect is in use with a "memory" database"